integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionLockContentionTest.java

```diff
@@ -324,32 +324,12 @@ private void ensureCompactionTaskCount(int expectedCount)
    */
   private int getNumberOfCompletedCompactionTasks()
   {
-    List<TaskResponseObject> incompleteTasks = indexer
-        .getUncompletedTasksForDataSource(fullDatasourceName);
     List<TaskResponseObject> completeTasks = indexer
         .getCompleteTasksForDataSource(fullDatasourceName);
 
-    printTasks(incompleteTasks, "Incomplete");
-    printTasks(completeTasks, "Complete");
-
     return (int) completeTasks.stream().filter(this::isCompactionTask).count();
   }
 
-  private void printTasks(List<TaskResponseObject> tasks, String taskState)
```

Comment:
Are these not helpful from a debugging point of view? 🤔

Reply:
Not really, it just keeps printing a bunch of unnecessary task IDs, some of which might not even be relevant.
If something does fail, this info does not come in handy either.